src/silly_sprites/opengl/window.hpp

```diff
@@ -26,7 +26,7 @@ namespace sly::gl {
         [[nodiscard]] GLFWwindow* get() const;
 
     public:
-        [[nodiscard]] static tl::expected<Window, GlError> create(int width, int height);
+        [[nodiscard]] static Window create(int width, int height);
```

Comment:
I like that you got rid of the `tl::expected` here (like we talked about), but it makes no sense anymore now to have this as a static factory function. Just move the code into the constructor and let it throw if something goes wrong.